apps/console/src/public/deployment.config.json

```diff
@@ -692,7 +692,9 @@
                         "console:loginAndRegistration"
                     ],
                     "read": [
-                        "internal_governance_view"
+                        "internal_governance_view",
```

Comment:
Let's link the framework PRs here for adding scopes and updating deployment config.

Reply:
https://github.com/wso2/carbon-identity-framework/pull/5859